### PR TITLE
Add user detail page

### PR DESCRIPTION
**Pull request by whDongRui**
* Modify table relative query methods

### CODE REVIEW

File: src/stores/app/version.js [hunk 1]
```diff
@@ -13,27 +13,31 @@ export default class AppVersionStore extends Store {
   @observable dialogType = ''; // delete, show_all
   @observable readme = '';
   @observable totalCount = 0;
-  @observable currentPage = 1;
+
+  @observable currentPage = 1; //version table query params
   @observable searchWord = '';
+  @observable defaultStatus = ['active'];
```
Comment (iwisunny) [inline src/stores/app/version.js]: No need to set this prop as observable

File: src/stores/cluster/index.js [hunk 2]
```diff
@@ -16,14 +16,18 @@ export default class ClusterStore extends Store {
...
   @observable isModalOpen = false;
+  @observable modalType = '';
 
   @observable clusterId; // current delete cluster_id
   @observable operateType = '';
 
-  @observable currentPage = 1;
+  @observable currentPage = 1; //cluster table query params
   @observable searchWord = '';
+  @observable defaultStatus = ['active', 'stopped', 'ceased', 'pending', 'suspended'];
```
Comment (iwisunny) [inline src/stores/cluster/index.js]: `defaultStatus` no need to be set as observable

File: src/utils/columns/cluster-columns.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import { I18n } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import TdName from 'components/TdName';
+import Status from 'components/Status';
+import TimeShow from 'components/TimeShow';
+import { getObjName } from 'utils';
+import { ProviderName } from 'components/TdName';
+
+export default function(runtimes, versions) {
```
Comment (iwisunny) [inline src/utils/columns/cluster-columns.js]: refactor to es6 arrow function

File: src/pages/Admin/Apps/Detail/index.jsx [hunk 4]
```diff
@@ -32,16 +32,36 @@ export default class AppDetail extends Component {
...
-  }
+  changeDetailTab = async tab => {
+    const { appStore, clusterStore, appVersionStore, runtimeStore, match } = this.props;
+    const { appId } = match.params;
+    appStore.detailTab = tab;
+
+    if (tab === 'Clusters') {
+      clusterStore.loadPageInit();
+      clusterStore.appId = appId;
+      await clusterStore.fetchAll({ status: 'deleted' });
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Admin/Apps/Detail/index.jsx]: test clusters data, modify: await clusterStore.fetchAll();

File: src/pages/Admin/Apps/Detail/index.jsx [hunk 5]
```diff
@@ -32,16 +32,36 @@ export default class AppDetail extends Component {
...
+          status: ['active', 'deleted'],
+          runtime_id: runtimeIds
+        });
+        await appVersionStore.fetchAll({
+          status: ['active', 'deleted'],
+          version_id: versionIds
+        });
+      }
+    } else if (tab === 'Versions') {
+      appVersionStore.loadPageInit();
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Admin/Apps/Detail/index.jsx]: ok, delete loadPageInit when change tab.

File: src/stores/app/version.js (hunk 1 above)
Comment (whDongRui) [reply src/stores/app/version.js]: ok, delete @observable.

File: src/stores/cluster/index.js (hunk 2 above)
Comment (whDongRui) [reply src/stores/cluster/index.js]: ok, delete @observable .

File: src/pages/Admin/Repos/Detail/index.jsx [hunk 6]
```diff
@@ -121,62 +122,21 @@ export default class RepoDetail extends Component {
       if (runtimes.length > 0) {
         const runtimeIds = runtimes.map(item => item.runtime_id);
         await clusterStore.fetchAll({
-          //status: ['active', 'stopped', 'ceased', 'pending', 'suspended', 'deleted'],
-          runtime_id: runtimeIds,
-          limit: 200
+          runtimeid: runtimeIds,
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Admin/Repos/Detail/index.jsx]: ok

**Pull request merged**